pages/lib/charts_data_explorer.py

```diff
@@ -43,8 +44,8 @@ def custom_heatmap(df, global_local, var, time_filter_info, data_filter_info, si
         range_z = var_range
     else:
         # Set maximum and minimum according to data
-        data_max = 5 * ceil(df[var].max() / 5)
-        data_min = 5 * floor(df[var].min() / 5)
+        data_max = get_data_max(df[var])
```

Comment:
can you please use a more meaningful name for the function?

Reply:
if this function is calculating the max value it should be called `get_max_value`. You should also combine this function with `get_data_min` and for example call the function `get_max_min_value`